### PR TITLE
Misc. editor_fixes typos

### PR DESCRIPTION
**Pull request by luzpaz**
Most of these are trivial comment typos. but some are doxy and even variables (i usually don't fix these bug i searched and found that in certain cases i could) that were misspelled.
Remaining 2 typos I didn't know what to do about:
```
$ codespell -d -q 3 --skip="*.po,*.ts"
./PythonDebuggerView.cpp:465: dont  ==> don't
./PythonCode.h:247: ot  ==> to
```

### CODE REVIEW

File: src/Gui/PythonEditor.cpp [hunk 1]
```diff
@@ -1343,7 +1343,7 @@ bool PythonEditorCodeAnalyzer::keyPressed(QKeyEvent *e)
 
     bool bailout = false;
 
-    // do nothing if ctrl or shift on ther own
```
Comment (luzpaz) [inline src/Gui/PythonEditor.cpp]: This should be `Qt` instread of `Ot` right?

File: src/Gui/PythonEditor.cpp [hunk 2]
```diff
@@ -1466,9 +1466,9 @@ bool PythonEditorCodeAnalyzer::afterChoiceInserted(JediBaseDefinitionObj *obj, i
         return false;
 
     QString type = obj->type();
-    bool paranthesis = false;
```
Comment (luzpaz) [inline src/Gui/PythonEditor.cpp]: i search all `paranthesis` and found 3 in the whole branch all in the same file so I fixed them. Hope that's OK with you.

File: src/Gui/PythonEditor.cpp [hunk 3]
```diff
@@ -1310,7 +1310,7 @@ bool PythonEditorCodeAnalyzer::eventFilter(QObject *obj, QEvent *event)
         return true;
 
     if  (obj == d->editor) {
-        // Ot doesnt let me snatch keypress originating from Completer
```
Comment (luzpaz) [inline src/Gui/PythonEditor.cpp]: This should be `Qt` instread of `Ot` right?

Comment (mumme74) [inline src/Gui/PythonEditor.cpp]: Correct, good catch!

File: src/Gui/PythonEditor.cpp (hunk 2 above)
Comment (mumme74) [inline src/Gui/PythonEditor.cpp]: Of course it ok, im thankfull for all help!

**Comment by mumme74** [comment]
Thank you for looking into this!
I also pushed a fix for the 2 remaining ones you detected in PythonCode.h and PythonDebuggerView.cpp

**Comment by luzpaz** [comment]
My pleasure. Thanks again for all your work on this!

**Pull request merged**